src/inference/src/dev/icompiled_model.cpp

```diff
@@ -144,3 +149,26 @@ ov::SoPtr<ov::IRemoteContext> ov::ICompiledModel::get_context() const {
         return m_context;
     return m_plugin->get_default_context({});
 }
+
+bool ov::ICompiledModel::supports_caching() const {
+    bool supported(false);
+    try {
+        auto supportedMetricKeys = get_property(METRIC_KEY(SUPPORTED_METRICS)).as<std::vector<std::string>>();
+        supported = util::contains(supportedMetricKeys, METRIC_KEY(IMPORT_EXPORT_SUPPORT)) &&
+                    get_property(METRIC_KEY(IMPORT_EXPORT_SUPPORT)).as<bool>();
+    } catch (const InferenceEngine::Exception& ex) {
+    } catch (const ov::Exception& ex) {
+    }
+    if (!supported) {
+        supported = util::contains(get_property(ov::supported_properties.name()).as<std::vector<PropertyName>>(),
+                                   ov::device::capabilities) &&
+                    util::contains(get_property(ov::device::capabilities.name()).as<std::vector<std::string>>(),
+                                   ov::device::capability::EXPORT_IMPORT);
```

Comment:
It is property of plugin, not compiled model, is not it?

Reply:
Yes, looks like it doesn't work for the compiled model... So I will revert the logic with an exception in this case.

Reply:
Let's do this work step by step